tests/async_engine/test_api_server.py

```diff
@@ -8,13 +8,14 @@
 import requests
 
 
-def _query_server(prompt: str) -> dict:
+def _query_server(prompt: str, params=None) -> dict:
```

Comment:
Style nit:
```suggestion
def _query_server(prompt: str, params=None) -> dict:
    if params is None:
        params = {}
```

Reply:
Applied the change to the code, but this comment does not go away on GitHub. Marked it resolved.

Reply:
@zhuohan123 I applied this change. Check it. But GitHub did not close this comment for some reason.